bin/rm

```diff
@@ -45,17 +45,20 @@ use strict;
 use Getopt::Std;
 
 # Command line arguments, and other variables.
-use vars qw( $opt_i $opt_f $opt_r $opt_R $opt_P );
+use vars qw( $opt_i $opt_f $opt_r $opt_R );
 
-# Get the options.
-getOptions();
-
-#
-# Process each file named on the command line.
+getopts('ifrR') or showUsage();
+unless (@ARGV)
+{
+    warn "$0: missing argument\n";
+    exit 1;
+}
+my $Err = 0;
```

Comment:
Maybe the return value of processFile should decide if something gets added to $Err? I'd rather have that then subroutines accessing variables outside their scope.